docs/reference/transform/apis/get-transform-stats.asciidoc

```diff
@@ -72,9 +72,81 @@ include::{docdir}/rest-api/common-parms.asciidoc[tag=size-transforms]
 [[get-transform-stats-response]]
 ==== {api-response-body-title}
 
-`transforms`::
-  (array) An array of statistics objects for {transforms}, which are
-  sorted by the `id` value in ascending order.
+The API returns an array of statistics objects for {transforms}, which are
+sorted by the `id` value in ascending order. All of these properties are
+informational; you cannot update their values.
+
+`checkpointing`::
+(object) Contains statistics about <<transform-checkpoints,checkpoints>>.
+`checkpointing`.`changes_last_detected_at`:::
+(date) The timestamp when changes were last detected in the source indices.
+`checkpointing`.`last`:::
+(object) Contains statistics about the last completed checkpoint. 
+`checkpointing`.`last`.`checkpoint`::::
+(TBD) A unique identifier for the checkpoint.
+`checkpointing`.`last`.`time_upper_bound_millis`::::
+(date) TBD
+`checkpointing`.`last`.`timestamp_millis`::::
+(date) TBD
+
```

Comment:
checkpointing`.`next`:::
optional (object) Contains statistics about the next - currently in progress - checkpoint. This object only appears if the transform is currently processing data and only for the 1st checkpoint

It uses the same fields at last but has one more object:

`checkpoint_progress`::
(object) Contains statistics about the progress of the checkpoint. 

Not sure how much we want to go into the details, the inner fields are:
 - total_docs
 - docs_remaining
 - percent_complete
 - docs_processed
 - docs_indexed

This information is only available for batch transforms and for the 1st checkpoint of a continuous transform.